### PR TITLE
cli,test: Add --all option to scale command

### PR DESCRIPTION
**Pull request by jvatic**
#1664

### CODE REVIEW

File: cli/scale.go [hunk 1]
```diff
@@ -23,12 +23,20 @@ Ommitting the arguments will show the current scale.
...
 
 	$ flynn scale
 	web=4 worker=2
 
+	$ flynn scale --all
+	496d6e74-9db9-4cff-bcce-a3b44015907a
+	web=1 worker=2
+
+	632cd907-85ab-4e53-90d0-84635650ec9a
+	web=2
```
Comment (titanous) [inline cli/scale.go]: The current release (if any) should be marked.

File: cli/scale.go [hunk 2]
```diff
@@ -144,6 +159,64 @@ func runScale(args *docopt.Args, client *controller.Client) error {
...
+
+	formations := make([]*ct.Formation, len(releases))
+	for i, r := range releases {
+		formation, err := client.GetFormation(app, r.ID)
+		if err == controller.ErrNotFound {
+			formation = &ct.Formation{
+				AppID:     app,
+				ReleaseID: r.ID,
+				Processes: make(map[string]int),
+			}
```
Comment (titanous) [inline cli/scale.go]: Can we `append` with allocated capacity instead of putting blank releases in?

**Comment by jvatic** [comment]
@titanous comment addressed, also added skipping releases where nothing to show

File: cli/scale.go (hunk 2 above)
Comment (jvatic) [inline cli/scale.go]: That will make iteration below more difficult

**Comment by titanous** [comment]
LGTM

File: cli/scale.go (hunk 2 above)
Comment (jvatic) [inline cli/scale.go]: I suppose we could make `formations` a `map[string]*ct.Formation` where the release id is the lookup key

Comment (titanous) [inline cli/scale.go]: Yeah, that's a good idea.

**Pull request merged**